kinto/views/__init__.py

```diff
@@ -0,0 +1,26 @@
+import random
+import string
+
+from cliquet.storage import generators, exceptions
+from pyramid import httpexceptions
+
+
+class NameGenerator(generators.Generator):
+    def __call__(self):
+        try:
+            ascii_letters = string.ascii_letters
+        except ImportError:
+            ascii_letters = string.letters  # NOQA
+        alphabet = ascii_letters + string.digits + '-'
+        letters = [random.choice(alphabet) for x in range(8)]
+        return ''.join(letters)
```

Comment:
Note: this is not collision-proof. IHMO it shouldn't prevent merging, but could deserve an enhancement ticket :)